Model/Category.php

```
@@ -270,7 +270,7 @@ public function getParent()
     /**
      * {@inheritdoc}
      */
-    public function setMedia(MediaInterface $media = null)
+    public function setMedia($media = null)
```

You still need to raise an exception if the class is invalid